rest_framework-stubs/generics.pyi

```diff
@@ -27,12 +28,12 @@ class GenericAPIView(views.APIView, UsesQuerySet[_MT_co]):
     pagination_class: Optional[Type[BasePagination]] = ...
     def get_object(self) -> _MT_co: ...
     def get_serializer(self, *args: Any, **kwargs: Any) -> BaseSerializer[_MT_co]: ...
-    def get_serializer_class(self) -> BaseSerializer[_MT_co]: ...
+    def get_serializer_class(self) -> Type[BaseSerializer[_MT_co]]: ...
```

Comment:
does this method in fact return a class ? or does it expect a serializer instance?

Reply:
class